src/style.css

```
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.conteiner {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: rgba(255,255,255,1);
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-size: 16px;
+  border-radius: 0.5px;
+}
+
+.stars--1 :first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-of-type(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-of-type(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-of-type(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-of-type(n) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-of-type(-n+2),
.stars--3 :nth-of-type(-n+3),
.stars--4 :nth-of-type(-n+4),
.stars--5 :nth-of-type(n) {
  background-image: url(./images/star-active.svg);
}
```

давай скомбінуємо селектори